<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmakeProjectManager::Internal::LibraryDetailsWidget</class>
 <widget class="QWidget" name="QmakeProjectManager::Internal::LibraryDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>370</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="libraryLabel">
       <property name="text">
        <string>Library:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="libraryComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="libraryFileLabel">
       <property name="text">
        <string>Library file:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Utils::PathChooser" name="libraryPathChooser" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="includeLabel">
       <property name="text">
        <string>Include path:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Utils::PathChooser" name="includePathChooser" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="packageLabel">
       <property name="text">
        <string>Package:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="packageLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="platformGroupBox">
       <property name="title">
        <string>Platform</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="linCheckBox">
          <property name="text">
           <string>Linux</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="macCheckBox">
          <property name="text">
           <string>Mac</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="winCheckBox">
          <property name="text">
           <string>Windows</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="linkageGroupBox">
       <property name="title">
        <string>Linkage:</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="dynamicRadio">
          <property name="text">
           <string>Dynamic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="staticRadio">
          <property name="text">
           <string>Static</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="macGroupBox">
       <property name="title">
        <string>Mac:</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="libraryRadio">
          <property name="text">
           <string>Library</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="frameworkRadio">
          <property name="text">
           <string>Framework</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="winGroupBox">
       <property name="title">
        <string>Windows:</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="useSubfoldersCheckBox">
          <property name="text">
           <string>Library inside &quot;debug&quot; or &quot;release&quot; subfolder</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="addSuffixCheckBox">
          <property name="text">
           <string>Add &quot;d&quot; suffix for debug version</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="removeSuffixCheckBox">
          <property name="text">
           <string>Remove &quot;d&quot; suffix for release version</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>55</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
